migrations/v3.js

```diff
@@ -0,0 +1,133 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+const getCourse = content => {
+  const [course] = content.filter(({ _type }) => _type === 'course');
+  return course;
+};
+
+const getGlobals = content => {
+  return getCourse(content)?._globals?._extensions?._glossary;
+};
+
+describe('Glossary - v2.1.3 to v3.0.0', async () => {
+  // https://github.com/adaptlearning/adapt-contrib-glossary/compare/v2.1.3..v3.0.0
+
+  let course, courseGlossaryGlobals;
+
+  whereFromPlugin('Glossary - from v2.1.3', { name: 'adapt-contrib-glossary', version: '<3.0.0' });
+
+  whereContent('Glossary - where configured', async (content) => {
+    course = getCourse(content);
+    if (course._glossary) return true;
+  });
+
+  mutateContent('Glossary - add globals if missing', async (content) => {
+    courseGlossaryGlobals = getGlobals(content);
+    if (courseGlossaryGlobals) return true;
+    course._globals._extensions = course._globals._extensions || {};
+    courseGlossaryGlobals = course._globals._extensions._glossary = {};
+    return true;
+  });
+
+  mutateContent('Glossary - add new globals', async (content) => {
+    courseGlossaryGlobals.labelLink = 'Terms beginning with';
+    courseGlossaryGlobals.labelNavigation = 'Glossary navigation';
+    return true;
+  });
+
+  mutateContent('Glossary - change default title', async (content) => {
+    if (course._glossary.title !== '') return true;
+    course._glossary.title = 'Glossary';
+    return true;
+  });
+
+  mutateContent('Glossary - change default description', async (content) => {
+    if (course._glossary.description !== '') return true;
+    course._glossary.description = 'Select here to view the glossary for this course';
+    return true;
```

Comment:
Think we can condense this slightly, thoughts? 

```suggestion
    if (course._glossary.description === '') course._glossary.description = 'Select here to view the glossary for this course';
    return true;
```